packages/autocomplete-core/src/__tests__/getSources.test.ts

```diff
@@ -37,7 +41,98 @@ describe('getSources', () => {
     });
   });
 
-  test.todo('provides default source implementations');
+  test('provides a default source implementations', async () => {
+    const onStateChange = jest.fn();
+    const sources = createSource();
+    const getSources = () => {
+      return [
+        {
+          ...sources,
+          templates: {
+            item() {},
+          },
+        },
+      ];
+    };
+    const { inputElement } = createPlayground(createAutocomplete, {
+      getSources,
+      onStateChange,
+    });
+
+    inputElement.focus();
+    userEvent.type(inputElement, 'a');
+
+    await runAllMicroTasks();
+
+    expect(onStateChange).toHaveBeenCalledWith({
+      prevState: expect.anything(),
+      state: expect.objectContaining({
+        collections: expect.arrayContaining([
+          expect.objectContaining({
+            source: expect.objectContaining({
+              getItemInputValue: sources.getItemInputValue,
+              getItemUrl: sources.getItemUrl,
+              getItems: sources.getItems,
+              onActive: expect.any(Function),
+              onSelect: expect.any(Function),
+              templates: expect.objectContaining({
+                item: expect.any(Function),
+              }),
+            }),
+          }),
+        ]),
+      }),
+    });
+  });
+
+  test('concat getSources from plugins', async () => {
+    const sources = createSource();
+    const getSources = () => {
+      return [
+        {
+          ...sources,
+          templates: {
+            item() {},
+          },
+        },
+      ];
+    };
+    const plugin = {
+      getSources: jest.fn((..._args: any[]) => {
+        return [sources];
+      }),
+    };
 
-  test.todo('concat getSources from plugins');
+    const onStateChange = jest.fn();
+    const { inputElement } = createPlayground(createAutocomplete, {
+      onStateChange,
+      getSources,
+      plugins: [plugin],
+    });
+
+    inputElement.focus();
+    userEvent.type(inputElement, 'a');
+
+    await runAllMicroTasks();
+
+    expect(onStateChange).toHaveBeenCalledWith({
+      prevState: expect.anything(),
+      state: expect.objectContaining({
+        collections: expect.arrayContaining([
```

Comment:
We want to check that there are now 2 collections (1 from top-level and 1 from plugin). We don't need to check the default implem here.